examples/sys_sysmon/sysmon-client.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+import base64
+import binascii
+from pathlib import Path
+
+from google import protobuf
+from google.protobuf import json_format
+from serial import Serial
+import typer
+
+from PB_Sysmon_pb2 import PB_Sysmon
+
+serial_port = Serial()
+
+reset_uuid = 3351980141
```

Comment:
Reset is not used, you can simplify the example files (*.cpp, *.py, *.md) by removing it.

Reply:
done